graphql_compiler/schema_transformation/rename_query.py

```diff
@@ -27,14 +31,15 @@ def rename_query(
     any occurrences of type Y in the input query ast will be renamed to X.
 
     All type names (including ones in type coercions), as well as root vertex fields (fields
-    of the query type) will be renamed. No other field names will be renamed.
+    of the query type) will be renamed.
 
     Args:
         ast: represents a query
         renamed_schema_descriptor: namedtuple including the attribute reverse_name_map, which maps
-                                   the new, renamed names of types to their original names. This
-                                   function will revert these renamed types in the query ast back to
-                                   their original names
+                                   the new, renamed names of types to their original names, and
+                                   reverse_field_name_map which has a similar role for fields. This
+                                   function will revert these renamed types/ fields in the query ast
```

Comment:
nit: AST is an acronym, so let's try to consistently capitalize it here and elsewhere, and especially in end-user-facing messages and docstrings. This function is going to be part of the public API, so the quality bar should be particularly high.
```suggestion
                                   function will revert these renamed types/ fields in the query AST
```